djangoProject/users/views.py

```diff
@@ -21,6 +21,21 @@ class UserRegistrationView(generics.CreateAPIView):
     serializer_class = UserRegistrationSerializer
     permission_classes = [AllowAny]
 
+    def create(self, request, *args, **kwargs):
+        # Serialize and validate the data
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        # Save the user
+        user = serializer.save()
+        role = user.user_type
+
+        # Redirect based on the role
+        if role == "startup" or role == "both":
+            return HttpResponseRedirect(f"/profiles/start-up/create/?id_user={user.user_id}")
+
+        return HttpResponseRedirect("/")
```

Comment:
`HttpResponseRedirect` is more suitable for browser-based redirects. Since this is an API view, you should return a proper HTTP response (e.g., `Response`) with a URL that the frontend can use to redirect the user.